components/menteeList/MenteeListSection.js

```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react';
 import {
   ScrollView, View, Text, StyleSheet, Image
 } from 'react-native';
-import { Button } from '../common/Button';
+import Button from '../common/Button';
+import AvatarSource from '../../assets/img_avatar.png';
```

Comment:
It's probably out of scope for this PR, but I don't know if we need this anymore since we now collect the image from Facebook.

Reply:
yeah that has been addressed in the clean out PR I think